### PR TITLE
feat [GRO-737]: redirect how-auctions-work to a support link 

### PR DESCRIPTION
**Pull request by TMcMeans**
### The type of this PR is: feat

This PR solves [GRO-737]

### Description

As part of the v3 static page migration, we are retiring the original `how-auctions-work` page rather than implementing any design updates and re-directing instead to this support link: https://support.artsy.net/hc/en-us/articles/4419870291351-The-Complete-Guide-to-Auctions-on-Artsy

[GRO-737]: https://artsyproduct.atlassian.net/browse/GRO-737?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TMcMeans** [comment]
Starting the process off by removing the how-auctions-work apps from desktop and mobile. Making small, incremental commits since deleting whole folders and stuff

**Comment by TMcMeans** [comment]
all the old code is pretty much gone, which just leaves removing the newer `HowAuctionsWork2` app code and any routing since the page is being retired, then adding the redirect.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cb1133d5ad46732619e4d4d23f8bffa77e31ac51
</p>

**Comment by TMcMeans** [comment]
is an integration test necessary here??

File: src/lib/middleware/hardcodedRedirects.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ const redirects = {
   "/feature/art-fairs": "art-fairs",
   "/settings": "/user/edit",
   "/collector/edit": "/profile/edit",
+  "/how-auctions-work2":
```
Comment (damassi) [inline src/lib/middleware/hardcodedRedirects.ts:33]: We should remove the `2` here

**Review by damassi — DISMISSED** [review]
One tiny fix, otherwise looks great 👍 Thanks for the old code cleanup

**Comment by damassi** [comment]
🚀

**Pull request merged**